src/sentry/web/frontend/react_page.py

```diff
@@ -205,3 +205,10 @@ def handle(self, request: HttpRequest, organization, **kwargs) -> HttpResponse:
 class GenericReactPageView(BaseView, ReactMixin):
     def handle(self, request: HttpRequest, **kwargs) -> HttpResponse:
         return self.handle_react(request, **kwargs)
+
+
+class AuthV2ReactPageView(GenericReactPageView):
+    auth_required = False
+
+    def handle_auth_required(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        raise Exception("This should not be called")
```

Comment:
Will you need feature flag gating on this view as well?

Reply:
Probably not. I'm still thinking through flags to protect login/registration endpoints, and I'm leaning towards middleware + custom headers instead of flags.